person.rb

```diff
@@ -12,14 +13,19 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
     @name
   end
 
   def can_use_services?
-    of_age? || @parent_permission
+    of_age? || @parent_permission == true
+  end
+
+  def add_rental(book, date)
+    Rental.new(date, self, book)
```

Comment:
- [ ] Kindly pay attention to this arrangement of the arguments. `book` comes before `person` so we should have something like this:

  ```
  Rental.new(date, book, self)
  ```

Reply:
done